exatomic/va/va.py

```diff
@@ -502,7 +502,89 @@ def __init__(self, uni, delta_type=0, fdx=-1, *args, **kwargs):
         self.delta = gen_delta(freq, delta_type)
         self.disp = self._gen_displaced(freq, atom, fdx)
 
+class VAMeta(TypedMeta):
+    grad_0 = Gradient
+    grad_plus = Gradient
+    grad_minus = Gradient
+    gradient = Gradient
+
+class VA(metaclass=VAMeta):
```

Comment:
For py2 compatibility, you could use `six.with_metaclass(...)` (or something like that. That said, py2 is leaving Earth in a couple of years..